server/api/genre.js

```diff
@@ -0,0 +1,18 @@
+// const qs = require('qs');
+const {Genre} = require('../db/index')
```

Comment:
Model should be imported after package dependencies and again spacing w/ brackets

Reply:
Remove line 1, we don't need qs here.